share/validate-product-config/product-config-schema.rnc

```diff
@@ -176,15 +191,21 @@ ds.htmlinlinecontent =
 # ROOT
 
 ds.product =
+  ## Product (or top-level category, root element)
   element product {
     # This ID value can only be checked when we take all of the productconfig
     # files together (i.e. after we have run stitch.sh). In any case, it does
     # not make sense to use xsd:id here because there can only ever be one
     # of these ID per productconfig file.
+    ## Product ID, must be unique across entire site config, used as first-level path element
     attribute productid { ds.type.id },
+    ## Version of the schema, verifies that Docserv² major version and version of product configuration match
     attribute schemaversion { ds.schema.version },
+    ## Whether to enable this configuration file (`true` by default)
     attribute enabled { xsd:boolean }?,
```

Comment:
Well, `true` is the implicit default, because Docserv² treats absence of the attribute as it being true.

Is it useful to formally define this in the schema?

Reply:
Well, it would be useful to have a formal definition if you look at the RNC, use it in an editor that shows these things, or extract some documentation.

On the other hand, it's probably a minor thing as it's already integrated in the Docserv² code. I suppose, we can resolve that.